File: Problem solving with c++/Problem solving Level 2/Problem19.cpp
/*
Problem19 |===============================================

Write a program to print 3 random numbers from 1 to 10 

Output :
6
2
3
==========================================================
*/

// This is my code solution :

#include <iostream>
#include <cstdlib> 
using namespace std; 

int RandomNumber(int From, int To) 
{ 
//Function to generate a random number 
int randNum = rand() % (To - From + 1) + From; 
return randNum; 
} 
int main() { 
//Seeds the random number generator in C++, called only once 
    srand((unsigned)time(NULL)); 
    
    cout << RandomNumber(1, 10) << endl; 
    cout << RandomNumber(1, 10) << endl; 
    cout << RandomNumber(1, 10) << endl; 
return 0; 
} 

File: Problem solving with Python/Problem19.py
"""
Problem19 |===============================================

Write a program to print 3 random numbers from 1 to 10 

Output :
6
2
3
==========================================================
"""

import random

def random_number(start: int, end: int) -> int:
    """
    Generates a random number between 'start' and 'end' (inclusive).
    """
    return random.randint(start, end)

# Generate and print random numbers
print(random_number(1, 10))
print(random_number(1, 10))
print(random_number(1, 10))
